code/modules/research/designs/mecha/boards.dm

```diff
@@ -134,7 +134,7 @@
 	name = "Circuit Design (\"Marauder\" Central Control module)"
 	desc = "Allows for the construction of a \"Marauder\" Central Control module."
 	id = "marauder_main"
-	req_tech = list(Tc_PROGRAMMING = 4)
+	req_tech = list(Tc_PROGRAMMING = 4, NANOTRASEN = 5)
```

Comment:
I just fixed the nanotrasen research level to match the others, so you should change this to Tc_NANOTRASEN